### PR TITLE
perf: lazy load completion deps (for WASM)

### PR DESCRIPTION
**Pull request by akshayka**
Lazy load some dependencies so they don't block the notebook code from rendering.

### CODE REVIEW

File: frontend/src/core/wasm/worker/bootstrap.ts [hunk 1]
```diff
@@ -206,6 +206,10 @@ export class DefaultWasmController implements WasmController {
 
   private async loadNotebookDeps(code: string, foundImports: Set<string>) {
     const pyodide = this.requirePyodide;
+    // Additional dependencies of marimo that are lazily loaded.
+    //
+    // pyodide-http is a patch to make basic http requests work in pyodide
+    await pyodide.loadPackage(["docutils", "Pygments", "jedi", "pyodide-http"]);
```
Comment (mscolnick) [inline frontend/src/core/wasm/worker/bootstrap.ts:212]: Does this actually make it faster? This is now serial instead of parallel

Comment (mscolnick) [reply frontend/src/core/wasm/worker/bootstrap.ts:212]: Oh I see it shows the notebook code faster but not started. You can still push this down to be parallel with the users deps

Comment (akshayka) [reply frontend/src/core/wasm/worker/bootstrap.ts:212]: Yea, it makes the code show faster. Will do, thanks for the tip

Comment (mscolnick) [reply frontend/src/core/wasm/worker/bootstrap.ts:212]: You can promise.all this below at line 224

**Comment by mscolnick** [comment]
ill make the updates. pinning will help with perf

**Review by mscolnick — APPROVED** [review]

**Comment by mscolnick** [comment]
@akshayka - i was wrong about pinning versions. it doesn't matter since they are downloaded from pyodide custom wheels which are pinned by default

**Pull request merged**